about.html

```diff
@@ -0,0 +1,156 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Links Part -->
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="project icon" href="./assets/logos/Logo.png"  type="image/x-icon"
+    />
+    <title>Capstone_Project</title>
+</head>
+<body>
+    <header>
+    <!-- Header section -->
+    <div class="header">
+        <div class="quicklink social">
+            <a href="#"><img src="assets/logos/linkedinicon.png" alt="linkedin Icon" /></a>
+        </div>
+        <div class="quicklink social">
+            <a href="#"><img src="assets/logos/Twitter.png" alt="Twitter Icon" /></a>
+        </div>
+        <div class="quicklink">
+            <p><a href="#"></a>English</p> 
+        </div>
+        <div class="quicklink">
+            <p><a href="#"></a>My Page</p> 
+        </div>
+        <div class="quicklink">
+            <p><a href="#"></a>Logout</p> 
+        </div>
+    </div>
+    <!-- Navbar section -->
+     <div class="humberger">
+        <div class="line1"></div>
+        <div class="line2"></div>
+        <div class="line3"></div>
+     </div>
+     <div id="navbar" class="navbar">
+        <div class="logo">
+            <a href="index.html"><img src="assets/logos/Logo.png" alt="page logo" /></a>
+        </div>
+        <ul>
+            <li class="links home-page"><a href="index.html">Home</a></li>
+            <li class="links"><a href="about.html">About</a></li>
+            <li class="links"><a href="#">Program</a></li>
+            <li class="links"><a href="#">Join</a></li>
+            <li class="links"><a href="#">Sponsor</a></li>
+            <li class="links last-link"><a href="#">News</a></li>
+            <li class="links focused"><a href="#">CC Campaign</a></li>
+        </ul>
+     </div>
+</header> 
+<main>
+    <section id="about">
+      <p class="quick-link">
+        <a class="top-link" href="index.html">Home</a><span> &#62; </span>
+          <a class="top-link" href="about.html">About</a>
+      </p>
```

Comment:
Good job, but it's recommended to follow the original design idea. So please kindly make sure to update it's colors and move it to the right.